.yamllint

```diff
@@ -0,0 +1,10 @@
+
+rules:
+    # Too many long linux command that are tricky and become less readable

```

Comment:
But, you have changed the line lengths of many YAML files.

I suggest that you undo those changes, as they are unnecessary for this patch, and they are not enforced by CI.